extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -64,6 +64,13 @@ const loadMetricHandlers = jsutil.once(repConfig => {
         labelNames: ['origin', 'replicationStatus'],
     });
 
+    const replicationStatusDurationSeconds = ZenkoMetrics.createHistogram({
+        name: 'replication_status_process_duration_seconds',
+        help: 'Duration of replication status processing',
```

Comment:
Is this a duration or a latency? The ticket is about latency but here we use the word the duration.

Reply:
A latency is a duration, it basically depends on the point of view.
Here we are expected to measure 
* The average latency for operations → hence the "latency" in title
* Defined as "elapsed ms for the time when an entry is picked up from the Kafka to the time its status is successfully updated in MongoDB" → i.e. the duration of processing